### ISSUE TITLE
Multiple plots in one chunk, sometimes one gets left out

### ISSUE THREAD

**Issue by Deleetdk** [yihui/knitr#1346]
![knitr problem 4](https://cloud.githubusercontent.com/assets/6759913/21915481/5f0dedcc-d93a-11e6-83ba-94e1f6091700.png)

Also happens with ggplot2

![knitr problem 5](https://cloud.githubusercontent.com/assets/6759913/21915502/8c0e2166-d93a-11e6-8650-086d7949b3b9.png)

But not if one adds e.g. a `print `call first:

![knitr problem 6](https://cloud.githubusercontent.com/assets/6759913/21915527/aec30f28-d93a-11e6-8c27-cfa7736ae0d8.png)

**Comment by yihui** [comment]
Thanks for the report. I guess that is more likely to be an issue of the RStudio IDE (R Markdown notebooks), unless you can reproduce the issue when clicking the Knit button instead of executing the code chunks inline.

Cc @jmcphers @kevinushey

**Comment by Deleetdk** [comment]
The html output seems fine, but the notebook output is very bugged. Not only are many figures missing, large parts of code chunks are also missing. The code is part of a larger project, so I'm not sure how you want me to share the error producing code or the result.

**Comment by kevinushey** [comment]
This indeed looks like a bug on the IDE side; I'll file an issue on our internal tracker and we'll see what we can do. Thanks for reporting!

**Comment by kevinushey** [comment]
@Deleetdk: note that the output displayed for a rendered R Notebook file should match what you see in the IDE; if you have chunks that are un-executed then that state will be respected in the `.nb.html` document generated; ie, those chunks will have no outputs. Are you seeing behavior like this, or something else?

**Comment by jmcphers** [comment]
@Deleetdk What version of RStudio are you using?

**Comment by Deleetdk** [comment]
@jmcphers 

Using the latest version of everything:

RStudio: Version 1.0.136
All packages updated to latest CRAN.

```
R version 3.3.2 (2016-10-31)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows >= 8 x64 (build 9200)

locale:
[1] LC_COLLATE=Danish_Denmark.1252  LC_CTYPE=Danish_Denmark.1252    LC_MONETARY=Danish_Denmark.1252
[4] LC_NUMERIC=C                    LC_TIME=Danish_Denmark.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] tidyr_0.6.1          penalized_0.9-47     lubridate_1.6.0      kirkegaard_2017.01   magrittr_1.5        
 [6] forcats_0.1.1        assertthat_0.1       purrr_0.2.2          weights_0.85         mice_2.25           
[11] Rcpp_0.12.8          gdata_2.17.0         Hmisc_4.0-2          Formula_1.2-1        survival_2.40-1     
[16] lattice_0.20-34      stringr_1.1.0        polycor_0.7-10       psych_1.6.12         ggplot2_2.2.1       
[21] XLConnect_0.2-12     XLConnectJars_0.2-12 plyr_1.8.4           dplyr_0.5.0          pacman_0.4.1        

loaded via a namespace (and not attached):
 [1] mvtnorm_1.0-5       gtools_3.5.0        glmnet_2.0-5        rprojroot_1.1       digest_0.6.11      
 [6] foreach_1.4.3       R6_2.2.0            backports_1.0.4     acepack_1.4.1       evaluate_0.10      
[11] lazyeval_0.2.0      multilevel_2.6      data.table_1.10.0   rpart_4.1-10        Matrix_1.2-7.1     
[16] checkmate_1.8.2     rmarkdown_1.3       labeling_0.3        splines_3.3.2       foreign_0.8-67     
[21] munsell_0.4.3       base64enc_0.1-3     mnormt_1.5-5        htmltools_0.3.5     nnet_7.3-12        
[26] tibble_1.2          gridExtra_2.2.1     htmlTable_1.8       codetools_0.2-15    MASS_7.3-45        
[31] psychometric_2.2    grid_3.3.2          nlme_3.1-128        jsonlite_1.2        gtable_0.2.0       
[36] DBI_0.5-1           scales_0.4.1        stringi_1.1.2       latticeExtra_0.6-28 RColorBrewer_1.1-2 
[41] iterators_1.0.8     tools_3.3.2         rsconnect_0.7       parallel_3.3.2      yaml_2.1.14        
[46] colorspace_1.3-2    cluster_2.0.5       rJava_0.9-8         knitr_1.15.1       
```

**Comment by jmcphers** [comment]
Thanks! We think this a bug we've already fixed in the daily build. Would you be willing to give that a go to let us know if it resolves the issue?

https://dailies.rstudio.com/

**Comment by Deleetdk** [comment]
@kevinushey 

The Notebook output html error has apparently resolved itself. I don't understand it. I had restarted RStudio and cleared the cache multiple times and it still gave me incorrect output. But I did it again just now, and now it is fine. Weird!

@jmcphers 

Ah, so that's where you have been hiding the new versions. I had been checking [the preview page](https://www.rstudio.com/products/rstudio/download/preview/), but it has not been updated since mid December 2016.

Updating to the daily version fixed the plots going missing issue. Thanks!
